python/pyhail/dataset.py

```diff
@@ -266,6 +337,125 @@ def annotate_samples_list(self, input, root):
     def annotate_samples_table(self, input, sample_expr, root=None, code=None, config=None):
         """Annotate samples with delimited text file (text table).
 
+        **Examples**
+
+        >>> conf = pyhail.TextTableConfig(impute=True)
+        >>> vds1 = (hc.read('data/example1.vds')
+        >>>  .annotate_samples_table('data/samples1.tsv', 'Sample', root='sa.phenotypes', config=conf)
+        >>>  .print_schema())
```

Comment:
I'd used example1, 2, and 3 because the sample names were all different.  For now I've removed the numbers, and when it comes time to make these all runnable on the same example.vds, we can make a pass to harmonize all annotation files to be compatible.